src/Credentials/GCECredentials.php

```diff
@@ -493,7 +493,7 @@ public function fetchAuthToken(callable $httpHandler = null)
      */
     public function getCacheKey()
     {
-        return self::cacheKey;
+        return $this->tokenUri;
```

Comment:
Also add `/** @deprecated */` to the `cacheKey` constant

Reply:
Ahh I knew we needed to do something about it, but wasn't sure what. Let's add it.